s3tokenizer/model.py

```
@@ -25,7 +25,7 @@
 from einops import rearrange
 from torch import Tensor, nn
 
-from .utils import make_non_pad_mask, mask_to_bias, onnx2torch
+from .utils import make_non_pad_mask, mask_to_bias, onnx2torch, merge_tokenized_segments, padding
```

Ruff (0.11.9)

`.utils.padding` imported but unused

Remove unused import: `.utils.padding`

(F401)